docs/modules/usage/how-to/cli-mode.md

```diff
@@ -35,10 +35,10 @@ WORKSPACE_BASE=$(pwd)/workspace
 2. Set `LLM_MODEL` to the model you want to use:
 
 ```bash
-LLM_MODEL="claude-3-5-sonnet-20240620"
+LLM_MODEL="anthropic/claude-3-5-sonnet-20240620"
```

Comment:
Is this right? I couldn't test this specifically but for me use case I had to do:
openai/claude-3-5-sonnet@20240620

Reply:
Looks like you're right! There are probably other places we need to fix this